zulipterminal/ui_tools/boxes.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, view: Any) -> None:
         super(WriteBox, self).__init__(self.main_view(True))
         self.model = view.model
         self.view = view
+        self.msg_edit_id = False  # type: Optional[int]
```

Comment:
Should this be `None`? This seems like a mypy bug, assuming no type coercion or 'Falsey' type-checking? Are we using the latest mypy?

Reply:
Would it be better with `0` instead?

Reply:
I think that `None` makes it obvious that it is invalid, whereas `0` could be a valid id.

Reply:
Cool! Done.